SPECS/containerd2/containerd2.spec

```diff
@@ -0,0 +1,229 @@
+%global debug_package %{nil}
+%define upstream_name containerd
+%define commit_hash 207ad711eabd375a01713109a8a197d197ff6542
+
+Summary: Industry-standard container runtime
+Name: %{upstream_name}2
+Version: 2.0.0
+Release: 1%{?dist}
+License: ASL 2.0
+Group: Tools/Container
+URL: https://www.containerd.io
+Vendor: Microsoft Corporation
+Distribution: Azure Linux
+
+Source0: https://github.com/containerd/containerd/archive/v%{version}.tar.gz#/%{upstream_name}-%{version}.tar.gz
+Source1: containerd.service
+Source2: containerd.toml
+
+%{?systemd_requires}
+
+BuildRequires: git
+BuildRequires: golang < 1.23
+BuildRequires: go-md2man
+BuildRequires: make
+BuildRequires: systemd-rpm-macros
+
+Requires: runc >= 1.2.1
+
+# This package replaces the old name of moby-containerd
+Provides: moby-containerd = %{version}-%{release}
+Obsoletes: moby-containerd < %{version}-%{release}
```

Comment:
I would get rid of these moby-containerd provides/obsoletes. Users still calling containerd "moby-containerd" in their install scripts could unknowingly get this 2.0 package because of this.